src/quest_manager/templates/quest_manager/tab_quests_submission.html

```diff
@@ -8,81 +8,91 @@
+        <th class="col-sm-1 col-xs-2 text-right" data-field="xp" data-sortable="true">XP</th>
+        <th class="col-sm-2 hidden-xs" data-field="campaign" data-sortable="true">Campaign</th>
+        <th class="col-sm-1 hidden-xs text-center" data-field="tags" data-sortable="true">{{ config.custom_name_for_tag }}s</th>
+        <th class="col-sm-3 hidden-xs" data-field="status" data-sortable="true">Status</th>
+      </tr>
+    </thead>
+    <tbody role="tablist" aria-multiselectable="true">
+    {% for s in submissions %}
 
-  <div class="row panel-heading">
-    <div class="col-sm-1 col-xs-2 col-icon"></div>
-    <div class="col-sm-5 col-xs-8">Quest</div>
-    <div class="col-sm-1 col-xs-2 text-right">XP</div>
-    <div class="col-sm-2 hidden-xs text-center">Campaign</div>
-    <div class="col-xs-3 hidden-xs ">Status</div>
-  </div>
-  <div class="panel-group panel-group-packed"
-    id="accordian-{% if completed %}completed{% else %}inprogress{% endif %}"
-    role="tablist" aria-multiselectable="true">
-  {% for s in submissions %}
-    <div class="panel
-      {% if s.is_awaiting_approval %}panel-warning
-      {% elif s.is_returned %}panel-danger
-      {% elif s.do_not_grant_xp %}panel-muted
-      {% else %}panel-default
-      {% endif %}
-      accordian {% if s.id == active_id %} active {% endif %}">
-      <div class="panel-heading accordian accordian-trigger panel-heading-tall
-        {% if s.id == active_id %} active {% endif %}"
-        role="tab" id="heading-{{s.id}}"
+      <tr class="accordian-trigger panel-title
+        {% if s.is_awaiting_approval %}table-warning
+        {% elif s.is_returned %}table-danger
+        {% elif s.do_not_grant_xp %}table-muted
+        {% else %}table-default{% endif %}"
+        role="tab"
         aria-expanded="{% if s.id == active_id %}true{% else %}false{% endif %}"
-        aria-controls="collapse-{{s.id}}"
-        data-parent="#accordion-{% if completed %}completed{% else %}inprogress{% endif %}"
-        data-toggle="collapse" data-target="#collapse-{{s.id}}">
+        aria-controls="collapse-quest-{{s.id}}"
+        {% comment %} ID makes the ajax_content_loading.html file populate detailed content {% endcomment %}
+        id="heading-quest-{{s.id}}"
+      >
+        <td class="col-sm-1 col-xs-2 col-icon">
+          <img class="img-responsive panel-title-img img-rounded" src="{{ s.quest.get_icon_url }}" alt="icon"/>
+        </td>
+        <td class="col-sm-5 col-xs-8">{{s.quest_name}}</td>
+        <td class="col-sm-1 col-xs-2 text-center">
+          {% if s.quest.xp_can_be_entered_by_students %}{{s.xp_requested}}*{% else %}{{s.quest.xp}}{% endif %}
+        </td>
+        <td class="col-sm-2 hidden-xs"><small>
+          {% if s.quest.campaign %}{{ s.quest.campaign }} {% else %}-{% endif %}
+        </small></td>
+        <td class="col-sm-1 hidden-xs text-center"><small>{{s.quest.tags.all|join:", "}}</small></td>
```

Comment:
Can you add a default of "-" to tags, if their are none? This adds a nice visual indicator, similar to how Campaigns display, rather than whitespace.

Also, there is a cleaner way to do this, than what I did with campaigns: 
`{% if s.quest.campaign %}{{ s.quest.campaign }} {% else %}-{% endif %}`

I think can just be done like this:
`{{ s.quest.campaign|default:"-" }}

I think you can chain filters, since you're already using join, but I'm not sure which order you'll need to out them in.

Please update the campaign code too while you're in here!


![image](https://github.com/bytedeck/bytedeck/assets/10604391/9f4e302c-4c8a-47d5-ab0f-ce8b9763f50a)

Reply:
Done